micromamba/tests/test_env.py

```diff
@@ -484,3 +484,33 @@ def test_env_update_empty_base(tmp_home, tmp_root_prefix, tmp_path):
     packages = helpers.umamba_list("-p", env_prefix, "--json")
     assert any(package["name"] == "xtensor" for package in packages)
     assert any(package["name"] == "python" for package in packages)
+
+
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+@pytest.mark.parametrize("json_flag", [None, "--json"])
+def test_env_export_with_pip(tmp_home, tmp_root_prefix, tmp_path, json_flag):
+    env_name = "env-list_with_pip"
+    tmp_root_prefix / "envs" / env_name
+
+    env_file_yml = tmp_path / "test_env_yaml_content_to_install_numpy_with_pip.yaml"
+    env_file_yml.write_text(env_yaml_content_create_pip_pkg_with_version)
+
+    flags = list(filter(None, [json_flag]))
+    helpers.create("-n", env_name, *flags, no_dry_run=True)
+    helpers.install("-n", env_name, "-f", env_file_yml, *flags, no_dry_run=True)
```

Comment:
```suggestion
    helpers.create("-n", env_name, "-f", env_file_yml)
```

Couldn't we just have this simple command?
Also, I wouldn't use `*flags` here, because we care about parametrisation of `export` in this test, and it's flags are independent and may be aded later to check more cases

Reply:
Yes, this can be simplified. Some logic of old tests have been ported to other tests but this is unnecessary.